### ISSUE TITLE
rocket_contrib 0.3.9 fails to build on latest nightly 2018-05-03

### ISSUE THREAD

**Issue by berkus** [SergioBenitez/Rocket#632]
```
   Compiling rocket_contrib v0.3.9
error[E0557]: feature has been removed
 --> /Users/skarchebnyy/.cargo/registry/src/github.com-1ecc6299db9ec823/rocket_contrib-0.3.9/src/lib.rs:1:12
  |
1 | #![feature(macro_reexport)]
  |            ^^^^^^^^^^^^^^
  |
note: subsumed by `#![feature(use_extern_macros)]` and `pub use`
 --> /Users/skarchebnyy/.cargo/registry/src/github.com-1ecc6299db9ec823/rocket_contrib-0.3.9/src/lib.rs:1:12
  |
1 | #![feature(macro_reexport)]
  |            ^^^^^^^^^^^^^^

error[E0658]: The attribute `macro_reexport` is currently unknown to the compiler and may have meaning added to it in the future (see issue #29642)
  --> /Users/skarchebnyy/.cargo/registry/src/github.com-1ecc6299db9ec823/rocket_contrib-0.3.9/src/lib.rs:47:30
   |
47 | #[cfg_attr(feature = "json", macro_reexport(json_internal))]
   |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(custom_attribute)] to the crate attributes to enable

error: aborting due to 2 previous errors
```

**Comment by maxdeviant** [comment]
This is fixed by #624.

**Comment by SergioBenitez** [comment]
Duplicate of #626.
